tsconfig.json

```diff
@@ -1,10 +1,6 @@
 {
   "compilerOptions": {
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
+    "lib": ["dom", "dom.iterable", "esnext"],
```

Comment:
What's the rule for multi-line array and one-line array? I keep seeing those changes over and over...